packages/react/package.json

```diff
@@ -84,70 +84,69 @@
         "@types/classnames": "2.3.0",
         "@types/d3": "3.5.47",
         "@types/enzyme": "3.10.12",
-        "@types/enzyme-adapter-react-16": "1.0.6",
         "@types/escape-string-regexp": "1.0.0",
         "@types/faker": "4.1.12",
         "@types/jest": "27.4.1",
         "@types/jquery": "2.0.60",
         "@types/loadable__component": "5.13.4",
         "@types/lorem-ipsum": "2.0.0",
-        "@types/react": "16.14.25",
         "@types/react-bootstrap": "0.32.29",
         "@types/react-codemirror": "1.0.7",
         "@types/react-color": "3.0.6",
-        "@types/react-dom": "16.9.15",
+        "@types/react-dom": "17.0.15",
         "@types/react-infinite-scroll-component": "4.2.5",
         "@types/react-modal": "3.13.1",
         "@types/react-redux": "7.1.22",
         "@types/react-tether": "0.5.5",
         "@types/react-textarea-autosize": "4.3.6",
         "@types/react-transition-group": "2.9.2",
+        "@types/react": "17.0.44",
```

Comment:
The order of dependencies is still wrong 🙊 

"@types/react" comes before "@types/react-dom"

same for "@types/underscore" and "@types/underscore.string"

I guess you can just run `pnpm --recursive update @types/react@17.0.44` and see if pnpm fixes everything

Reply:
I sorted the lines with vscode and that's the order it chose 🤷